### ISSUE TITLE
remove deprecated twisted.runner.procutils

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5049]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5049|
|Type|task|
|Created|2011-04-12 19:00:55Z|
|Branch|https://github.com/twisted/twisted/tree/procutils-runner-5049|

[twisted/runner/procutils.py](twisted/runner/procutils.py) was deprecated 7 years ago (r11151), let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__5049 5049
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_procutils_runner_5049 branches/procutils-runner-5049
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1302634855000000 1302634855000000
changetime__1303140399000000 1303140399000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [31651]) Branching to 'procutils-runner-5049'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [31652]) removed module and added news file, refs [#5049](https://github.com/twisted/twisted/issues/5049)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Can you say when the module was deprecated, as recommended in ReviewProcess#Newsfiles? ReleaseRevisions suggests that this was Twisted 2.3 or earlier. Other than that, looks great, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [31694]) add reference to version number, refs [#5049](https://github.com/twisted/twisted/issues/5049)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [31695]) Merge procutils-runner-5049: Removed the twisted.runner.procutils module that was deprecated in Twisted 2.3.

Author: thijs
Reviewer: jesstess
Fixes: [#5049](https://github.com/twisted/twisted/issues/5049)
